sphinx/io.py

```diff
@@ -153,30 +152,30 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
 
 
-def read_doc(app: "Sphinx", env: BuildEnvironment, filename: str) -> nodes.document:
```

Comment:
I prefer to keep this as is and mark it deprecated. I'll send a change to your branch from now on.

Reply:
Done.